core/templates/domain/topics_and_skills_dashboard/topics-and-skills-dashboard-backend-api.service.ts

```diff
@@ -117,7 +117,7 @@ export class TopicsAndSkillsDashboardBackendApiService {
   private _topicsAndSkillsDashboardReinitializedEventEmitter =
     new EventEmitter<boolean>();
 
-  fetchDashboardData(): Promise<TopicsAndSkillDashboardData> {
+  async fetchDashboardDataAsync(): Promise<TopicsAndSkillDashboardData> {
```

Comment:
Hi @parulpriyedarshani, can you check where this funciton is used and make necessary changes there as well? Here are some files where I see this function being used:
![image](https://user-images.githubusercontent.com/11008603/100911948-e3e50100-34f5-11eb-8a76-76e51ee5d42d.png)

Can you please check and update usages for other functions you have modified in this PR? Thanks!

Reply:
@kevintab95 thanks for the review. I might have missed this by mistake. I will make the necessary changes asap!